helm-charts/docsum/values.yaml

```diff
@@ -103,6 +103,13 @@ docsum-ui:
 dashboard:
   prefix: "OPEA DocSum"
 
+# External LLM configuration
+externalLLM:
+  enabled: false
+  LLM_SERVICE_HOST_IP: "http://your-llm-server"
+  LLM_MODEL: "your-model"
+  OPENAI_API_KEY: "your-api-key"
+
```

Comment:
Please add default values for all the newly added conditional dependency in Chart.yaml